README.md

```diff
@@ -7,13 +7,27 @@ Ballerina Uuid Package
   [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/uuid.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fuuid)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-uuid/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-uuid)
 
-The `uuid` package is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+This library provides APIs to generate and inspect UUIDs (Universally Unique Identifiers).
 
-This package provides APIs to generate and inspect UUIDs (Universally unique identifier).
+The UUIDs are generated based on the [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122.html) standard. This module supports generating 4 versions of UUIDs.
 
-The UUIDs are generated based on the [RFC 4122](https://www.rfc-editor.org/rfc/rfc4122.html) standard. Other operations include validating a given UUID string and getting the version of a UUID string.
+### Version 1
 
-For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/swan-lake/learn/by-example/).
```

Comment:
We need to add this line in useful links section

Reply:
Updated in [dcd7453](https://github.com/ballerina-platform/module-ballerina-uuid/pull/362/commits/dcd74535d86bec7c7f1558727e4c328a69ea70d1)